yml/kube.yml

```diff
@@ -1,46 +1,46 @@
 kernel:
-  image: linuxkit/kernel:4.14.32
+  image: linuxkit/kernel:4.14.40
   cmdline: "console=tty0 console=ttyS0"
 init:
-  - linuxkit/init:be60dd1cb15ad39225512b6753304571d8c2fb17
-  - linuxkit/runc:1b67e997c720301c62052c8f29f5629ad751199b
-  - linuxkit/containerd:87ea9b027267b26bf5cc2d10a50c3eb9b5017df4
-  - linuxkit/ca-certificates:fb5c7bf842a330f5b47cdf71f950fe0c85f4a772
+  - linuxkit/init:c79d7587fcd0a195b8a3ecafe428a30e735cf2b4
```

Comment:
hmm, shouldnt these be the `linuxkit/init:v0.4` tagged versions?

Reply:
We decided some time ago to use the hashes not the tags here, I forget why exactly.

Reply:
It seems weird to do that...

Reply:
#53:
> Bump all hashes to those used in the v0.2 release. This uses the actual hashes rather than the tags since we aren't actually releasing this repository and I intend to keep tracking LinuxKit master, so using the tags would be temporary and perhaps misleading.

In any case I don't have cycles to rewrite the script to keep the tags at the moment.